README.md

```diff
@@ -24,3 +24,55 @@ The project deals with managing different layers of property on the basis of dif
 <img width="600" height="535" src="images/road_map.png"> 
   </p>
 
+## Contributors 🌟 
+
+Thanks goes to these wonderful people ✨✨:
+<table>
+	<tr>
+		<td align="center">
+			<a href="https://github.com/IIITM-Jay">
+				<img src="https://avatars2.githubusercontent.com/u/65283880?v=4" width="100px" alt="" />
+				<br /> <sub><b>IIITM-Jay</b></sub>
+			</a>
+			<br /> <a href="https://github.com/Jayshah6699/datascience-mashup/commits?author=IIITM-Jay">
```

Comment:
is the space under ```<br />``` is required...

Reply:
It is not required. If you want I can remove the spaces.

Reply:
I have removed the spaces @IIITM-Jay